python/mlcroissant/recipes/tfds_croissant_builder.ipynb

```diff
@@ -0,0 +1,1540 @@
+        "\n",
+        "TFDS has recently introduced a `CroissantBuilder`, which defines a TFDS dataset based on a Croissant 🥐 metadata file."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "9EcHnyZJ6wLA"
+      },
+      "source": [
+        "## Setup\n",
+        "\n"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "RAGp-2Qd962-"
+      },
+      "source": [
+        "Let's install and import the needed dependencies:"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "ByqeCJXM3GHf",
+        "outputId": "d7fbaaa8-a875-47fc-f6dc-cd8a28751ff6"
+      },
+      "outputs": [],
+      "source": [
+        "!pip install array_record\n",
+        "!pip install tfds-nightly\n",
+        "\n",
+        "!pip install mlcroissant@git+https://github.com/mlcommons/croissant#subdirectory=python/mlcroissant\n",
+        "\n",
+        "!pip install datasets\n",
+        "!pip install GitPython\n",
+        "!pip install Pillow\n",
+        "\n",
+        "import os\n",
+        "os.environ.pop('TFDS_DATA_DIR', None)\n",
+        "\n",
+        "import tensorflow_datasets as tfds\n",
+        "from tensorflow_datasets.core.dataset_builders.croissant_builder import CroissantBuilder"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "6W75CTzG7PoY"
+      },
+      "source": [
+        "To initialize a `CroissantBuilder` in TFDS, we need a Croissant 🥐 file describing a dataset.\n",
+        "\n",
+        "We can use the `from_huggingface_to_croissant` command of the `mlcroissant` library to convert a Hugging Face 🤗 datasets to Croissant JSON-LD files.\n",
+        "\n",
+        "Let's create a Croissant metadata file for [fashion_mnist](https://huggingface.co/datasets/fashion_mnist), a popular dataset for computer vision."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 3,
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "vqiO9xo_vqNP",
+        "outputId": "3ba1003a-a4d2-410c-e949-bcf15379d9cc"
+      },
+      "outputs": [
+        {
+          "name": "stdout",
+          "output_type": "stream",
+          "text": [
+            "Downloading builder script: 100% 4.83k/4.83k [00:00<00:00, 17.0MB/s]\n",
+            "Downloading metadata: 100% 3.13k/3.13k [00:00<00:00, 10.9MB/s]\n",
+            "Downloading readme: 100% 8.85k/8.85k [00:00<00:00, 23.6MB/s]\n",
+            "I1127 13:47:02.165981 140028632248320 from_huggingface_to_croissant.py:189] Done. Wrote Croissant JSON-LD to /tmp/croissant_1701092817.7705376.json\n"
+          ]
+        }
+      ],
+      "source": [
+        "!mlcroissant from_huggingface_to_croissant --dataset fashion_mnist"
```

Comment:
This script will be removed soon. Can you directly declare the Croissant file?

Reply:
Sure. I submitted the file under `datasets/`